tools/visualizations/vis_scheduler.py

```diff
@@ -214,8 +212,6 @@ def main():
                                 osp.splitext(osp.basename(args.config))[0])
 
     cfg.log_level = args.log_level
-    # register all modules in mmcls into the registries
-    register_all_modules()
```

Comment:
should here also replaced with `init_default_scope('mmcls')` ?

Reply:
No, because it uses runner, and the default scope will be initialized in the runner.